demo/test.c

```diff
@@ -1822,10 +1848,13 @@ static int test_s_mp_balance_mul(void)
    if ((e = mp_read_radix(&b, nb, 64)) != MP_OKAY) {
       goto LTM_ERR;
    }
+   kara = MP_KARATSUBA_MUL_CUTOFF;
+   MP_KARATSUBA_MUL_CUTOFF = b.used-2;
 
    if ((e = mp_mul(&a, &b, &c)) != MP_OKAY) {
```

Comment:
Could you please just call s_mp_balance_mul here (instead of changing the cutoffs)? This coverage issue you discovered is exactly what I meant, when I said the tests are not robust written like this.

And now that we have tommath_private.h included,  if we call s_mp_balance_mul directly, the issue is just gone :)

Reply:
> the issue is just gone

until we have proper symbol hiding activated and then we have an entirely new problem

Reply:
For now we don't have that issue. I will fix the building issues when the symbols are hidden, by doing a special test library with unhidden symbols. The test suite already accesses s_functions (jenkins).